app.cr

```diff
@@ -34,12 +34,13 @@ end
 
 def fetch_filter(context)
   filter = context.params["filter"]? || ""
-  filter.strip
+  filter = filter.strip.downcase unless filter.empty?
+  filter
 end
 
 private def matches_filter?(item: GithubRepo, filter: String)
-  item.name.includes?(filter) ||
-    (item.description && item.description.not_nil!.includes?(filter))
+  item.name.downcase.includes?(filter) ||
```

Comment:
Hum... I feel there's got to be a better way to structure this, but leave it as it is for now.